zingolib/src/wallet/describe.rs

```diff
@@ -204,7 +205,7 @@ impl LightWallet {
         }
     }
 
-    /// Return a copy of the blocks currently in the wallet, needed to process possible reorgs
+    /// Return a copy of the blocks current.maply in the wallet, needed to process possible reorgs
```

Comment:
Going by the rest of the changes in this file I'm guessing this is a search-and-replace artefact.